codec-http/src/test/java/io/netty/handler/codec/http/websocketx/WebSocketProtocolHandlerTest.java

```diff
@@ -45,6 +46,66 @@ public void testPingFrame() {
         assertFalse(channel.finish());
     }
 
+    @Test
+    public void testPingPongFlowControlWhenAutoReadIsDisabled() {
+        // Given
```

Comment:
remove comments as these not really provide any value.

Reply:
done